.github/workflows/README.md

```diff
@@ -85,4 +85,38 @@ jobs:
         uses: sportsball-ai/actions-hub/git-version-tag@<VERSION>
         with:
           version: ${{ needs.reusable_workflow_job.outputs.version }}
-```
\ No newline at end of file
+```
+
+# publish-to-confluence
+
+A configurable github action reusable workflow that will publish specified documents to Confluence.
+Outputs the confluence url of the published document(s) as an environment variable called $confluence_url.
+See https://github.com/justmiles/go-markdown2confluence for more details.
+
+## Example Usage
+
+```yaml
+name: publish README to confluence
+
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  publish_to_confluence:
+    if: github.event.pull_request.merged == true
+    uses: sportsball-ai/actions-hub/.github/workflows/publish-to-confluence.yml@v2
+    secrets: inherit
+    with:
+      comment: "this is version 1" # add an optional comment to the published page
```

Comment:
Can we make this comment not be about version? People are liable to try this without thinking.